src/main/java/org/sagebionetworks/bridge/validators/StudyParticipantValidator.java

```diff
@@ -130,6 +136,9 @@ public void validate(Object object, Errors errors) {
         for (String attributeName : participant.getAttributes().keySet()) {
             if (!app.getUserProfileAttributes().contains(attributeName)) {
                 errors.rejectValue("attributes", messageForSet(app.getUserProfileAttributes(), attributeName));
+            } else {
+                String attributeValue = participant.getAttributes().get(attributeName);
+                validateStringLength(errors, 255, attributeValue,"attributes["+attributeName+"]");
```

Comment:
Is this the appropriate format for the error field? Maybe adding quotes around the key would make it more clear.

Reply:
This is fine